agent/http_test.go

```diff
@@ -1087,9 +1087,9 @@ func TestHTTPServer_PProfHandlers_DisableDebugNoACLs(t *testing.T) {
 	req, _ := http.NewRequest("GET", "/debug/pprof/profile", nil)
 
 	httpServer := &HTTPHandlers{agent: a.Agent}
-	httpServer.handler(false).ServeHTTP(resp, req)
+	httpServer.handler().ServeHTTP(resp, req)
 
-	require.Equal(t, http.StatusNotFound, resp.Code)
+	require.Equal(t, http.StatusOK, resp.Code)
```

Comment:
is this a breaking change?

Reply:
I guess we can add a note of breaking change in the changelog. Here is an example: 

https://github.com/hashicorp/consul/pull/17415/files#diff-0687bd929af1acf8b825fd03984e85c2f561626921a76a9d150c8c1dc06933a5